install/helm/gloo/templates/1-gloo-deployment.yaml

```diff
@@ -273,6 +273,10 @@ spec:
           - name: HEADER_SECRET_REF_NS_MATCHES_US
             value: "true"
         {{- end}}
+        {{- if .Values.global.glooMtls.enabled }}
+          - name: GLOO_MTLS_SDS_ENABLED
```

Comment:
Would `GLOO_MTLS_ENABLED` be simpler ?

Reply:
Maybe, but trying to be consistent with whats already in place. For example https://github.com/solo-io/gloo/blob/main/install/helm/gloo/templates/1-gloo-deployment.yaml#L128